libc/newhdrgen/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+if(LLVM_LIBC_FULL_BUILD)
```

Comment:
if we've already checked for fullbuild when including this file we don't need to check it again here.